src/components/dialog-full-screen/dialog-full-screen.spec.js

```diff
@@ -67,13 +54,8 @@ describe('DialogFullScreen', () => {
       instance = wrapper.instance();
     });
 
-    it('renders a parent div with mainClasses attached', () => {
-      expect(wrapper.find('div.foo.carbon-dialog-full-screen').length).toEqual(1);
-    });
-
     it('renders the dialog', () => {
       expect(instance._dialog).toBeTruthy();
```

Comment:
Would this be better as a snapshot?

Reply:
It was in progress. I've updated unit tests. Thanks.

Reply:
Doesn't look like it's changed?

Reply:
Sorry, I didn't remove old test. I've just added new tests.